core-api/src/main/java/com/optimizely/ab/config/parser/JsonSimpleConfigParser.java

```diff
@@ -319,8 +317,7 @@ public ProjectConfig parseProjectConfig(@Nonnull String json) throws ConfigParse
             String id = (String)audienceObject.get("id");
             String key = (String)audienceObject.get("name");
             Object conditionObject = audienceObject.get("conditions");
-            JSONArray conditionJson = (JSONArray)conditionObject;
-            Condition conditions = ConditionUtils.<UserAttribute>parseConditions(UserAttribute.class, conditionJson);
+            Condition conditions = ConditionUtils.<UserAttribute>parseConditions(UserAttribute.class, conditionObject);
```

Comment:
Nice! I can't comment in `parseAudiences`, but don't we need to update it too? We want it to pass `conditionsObject` straight to `ConditionUtils.parseConditions` without converting to `JSONArray conditionJson`.